ndio/remote/neurodata.py

```diff
@@ -1172,6 +1196,7 @@ def create_channel(self, token, name, channel_type, dtype, readonly):
             raise ValueError("readonly must be 0 (False) or 1 (True).")
 
         # Good job! You supplied very nice arguments.
+        print self.url("{}/createChannel/".format(token))
```

Comment:
In general we try to not print anything to the user. This is just a style convention, but it makes the code more legible during runtime.